server/twilio.js

```diff
@@ -3,8 +3,8 @@
 
   module.exports = function(data) {
     var body = JSON.parse(data.Body);
-    var phone = body.reaction_fields.phone;
-    var text = body.action_fields.body || ''
+    var phone = '+1' + body.reaction_fields.phone;
```

Comment:
for validation isnt accounting for it having a +1 or not... but regardless that needs to be changed in the formulae repo not the twilio